src/bin/zmcontrol

```diff
@@ -386,15 +378,13 @@ sub doShutdown {
             next;
         }
         if ( $_ eq "mailbox" && !( -d "/opt/zextras/db/data" ) ) { next; }
-        if (   $_ eq "service"
-            || $_ eq "zimbra"
-            || $_ eq "zimbraAdmin"
-            || $_ eq "zimlet" )
+        if (   $_ eq "service" )
         {
             print "\tStopping $_ webapp...";
         }
         else {
-            print "\tStopping $_...";
+            my $service_name = renameServices($_);
+            print "\tStopping $service_name...";
```

Comment:
Instead of doing the "next" for zimbra, zimbraAdmin, zimlet,we can consider just not printing the result as follows:
```suggestion
            my $service_name = renameServices($_);
            if ($service_name ne "zimbra" ||  $service_name ne "zimbraAdmin" ||  $service_name ne "zimlet") {
              print "\tStopping $service_name...";
            }
```
Same for the start.

Reply:
We should do a map with something like `shouldPrint` here too... like for the name stuff, otherwise we end up with the same `if`s everywhere

Reply:
Yes, that's why I would have removed these services registration. I don't see why not print these when they are indeed still running through zmstorectl. I think it would have been acceptable to just have the "service webapp" that collects all 4 of them @keshavbhatt

Reply:
@frisonisland as mentined earlier, these are just indicators here in this case. But there is real logic attached to these service names in other components of Carbonio. hence, un-registering them will break that logic everywhere else.